sql-cli/sql_cli/__main__.py

```diff
@@ -55,19 +55,29 @@ def generate(
 
 
 @app.command(
-    help="Validate Airflow connection(s) provided in the configuration file for the given environment."
+    help="""
+    Validate Airflow connection(s) provided in the configuration file for the given environment.
+    """
 )
 def validate(
-    environment: str = typer.Argument(
+    project_dir: Optional[Path] = typer.Argument(
```

Comment:
```suggestion
    project_dir: Optional[Path] = typer.Option(
```

I think if we're going to make this optional it should be a named option instead of a an Argument that is dependent on order

Reply:
I have mixed feelings about this one - as it is, we are consistent with `flow init`.